### PR TITLE
Pull request for Blog app: add API endpoints

### PR DESCRIPTION
**Pull request by zuhebahmed88091**
Dear reviewer,

## In this PR:

---

### Project requirements

- Created an API endpoint to list all posts for a user.
posts

- Create an API endpoint to list all comments for a user's post.

comments

- Created an API endpoint to add a comment to a post. The owner of the comment is the user that makes it; it should **not** be set by passing a parameter.

addcomment1

- API endpoints should receive JSON and respond JSON as well.

---

Thank you!

Regards,

@zuhebahmed88091 @YoutLone

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -14,4 +14,13 @@
   # root 'users#index'
   # Defines the root path route ("/")
   # root "articles#index"
```
Comment (t-yanick) [inline config/routes.rb]: - [ ] Great job working on this milestone. To always keep our codebase professional, we have to remove commented lines of code. Kindly delete lines 14 and 16.

**Review by t-yanick — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED

Hi @zuhebahmed88091 and @YoutLone  , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Collaborative work. Commits from  both team members
:heavy_check_mark: Descriptive PR with screenshots
:heavy_check_mark: All API endpoints receive JSON and respond in a JSON format 
:heavy_check_mark: Use of correct gitflow
:heavy_check_mark: Good commit messages

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 1 above)
Comment (zuhebahmed88091) [reply config/routes.rb]: resolved

**Review by levy002 — APPROVED** [review]
## Hello @zuhebahmed88091 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter error :heavy_check_mark: 
- You implemented the suggested changes :heavy_check_mark: 

## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by zuhebahmed88091** [comment]
Thank you! @t-yanick @levy002

**Pull request merged**